azerothcore/azerothcore-wotlk
# fix(DB): SkillLine data imported from DBC

## Changes Proposed:
- Used Node DBC Reader to extract data from the DBC files.
- Updated the Database with the data extracted.

## Issues Addressed:


## SOURCE:


## Tests Performed:

Executed the test mentioned bellow.

## How to Test the Changes:

- Apply the SQL update file.
- On a new branch called pr-skillLine
- Run the command `git pull azerothcore-remote pull/6756/head`
- Run the command `git pull azerothcore-remote pull/9287/head`
- Compile and apply their respective update SQL files
- On any character, check for their skills such as 300/300 on their race's language or racial abilities.
- Report undesired behaviours.

## Known Issues and TODO List:


None to date.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
